### ISSUE TITLE
A11Y: Images/Video/GIFs buttons in 3P media tab are not announced as clickable

### ISSUE THREAD

**Issue by dreamofabear** [GoogleForCreators/web-stories-wp#4430]
## Bug Description

Images/Videos/GIFs buttons should have a clickable ARIA role like `button` or similar.

## Screenshots

<img width="467" alt="Screen Shot 2020-09-09 at 4 37 05 PM" src="https://user-images.githubusercontent.com/20525523/92651438-ddf6cf80-f2ba-11ea-96d1-e2369e5bf49e.png">

## Additional Context

- Plugin Version: 1.0.0-beta.2+fd68af7
- Operating System: macOS 10.15.6
- Browser: Chrome 85

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._
